src/utils.ts

```diff
@@ -0,0 +1 @@
+export const isTablet = layout => layout === "tablet" || layout === "phone";
```

Comment:
`layout !== 'desktop'`? Would be also more accurate to have `isDesktop` and do `!isDesktop(...)` or?

Reply:
yea, that's simpler. will update.

Reply:
@FlorianRappl https://github.com/openmrs/openmrs-esm-primary-navigation/pull/65/commits/5c9cc33dec48308a60f5fe331450239886ec77f2 updated to use isDesktop